### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by zxiiro**
Requires that Travis CI is first setup for the repo.

Signed-off-by: Thanh Ha <zxiiro@linux.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: java
+sudo: false
+script: mvn clean verify
```
Comment (vorburger) [inline .travis.yml]: we probably need an `-s ~/settings-odl.xml` with the settings-odl.xml here? 

Lemme set up a Travis on this repo right away...

Comment (zxiiro) [reply .travis.yml]: Yes, we'll need that if we have to pass it a custom settings.xml file.

**Comment by vorburger** [comment]
I'll merge this even though I don't think it will quite work yet, but (only, I think) that way I can enable Travis for this repo and we'll see...

**Comment by vorburger** [comment]
https://travis-ci.org/vorburger/opendaylight-bot/builds/373940060

**Pull request merged**